.github/workflows/gh-pages.yml

```diff
@@ -33,7 +33,9 @@ jobs:
       - name: Install dependencies
         run: poetry install --with docs
       - name: Build Docs
-        run: poetry run build-docs
+        run: HF_TOKEN="$HF_TOKEN" poetry run build-docs
+        env:
+          HF_TOKEN: "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"
```

Comment:
Do you really want this token to be publicly accessible rather than set as a repository secret and used as `${{ secrets.HF_TOKEN }}`?

Reply:
Oh, I see you already do this in a future PR, sorry for the noise

Reply:
It's just temporary to get things moving while I work with Neel to get it setup as a secret.